core/creators/themedui.js

```diff
@@ -384,6 +388,20 @@ CKEDITOR.replaceClass = 'ckeditor';
 		return editor;
 	}
 
+	function delayCreation( element, config, data, mode ) {
+		if ( config.delayDetached && element.isDetached() ) {
+			var intervalId = setInterval( function() {
+				if ( !element.isDetached() ) {
+					createInstance( element, config, data, mode );
+
+					clearInterval( intervalId );
+				}
+			}, 1000 );
```

Comment:
I guess this interval could be configurable, so one can adjust it depending on specific needs. And 1000 seems like a lot so I would go with default value with something between 50 - 100 ms.

Reply:
As we talked: I will add an additional callback. If user provides it, we call with editor creation function as an argument. In this way, user has more control over creation time and won't be affected by 'ticking' interval in background.  
  
If there will bo no callback: we fall back to our `setInterval` and we will set timer based on provided configuration or use the default value (50 - 100 ms) as @f1ames mentioned.

If there will be an incorrect callback provided - we use one of the standard errors described in the documentation.